src/lib/holocene/icon/paths.ts

```diff
@@ -22,6 +22,49 @@ const arrowUpIcon: Icon = {
   ],
 };
 
+const bookIcon: Icon = {
+  paths: [
+    {
+      d: 'M12 18.75V8.25C12 7.42157 12.6716 6.75 13.5 6.75H19.05C19.2985 6.75 19.5 6.95147 19.5 7.2V17.0357',
+      stroke: '#111827',
+      'stroke-width': '1.125',
+      'stroke-linecap': 'round',
```

Comment:
`stroke-linecap: round` is already set on the `<path />` in the component itself, so we don't need to specify it here.